README.md

```diff
@@ -17,7 +17,7 @@ cd MD-TASK
 ```
 *Install dependencies and set up Python virtual environment:*
 ```bash
-sudo apt-get install virtualenvwrapper python-dev libblas-dev liblapack-dev libatlas-base-dev gfortran libpng12-dev libfreetype6-dev python-tk r-base
+sudo apt-get install virtualenvwrapper python-dev libblas-dev liblapack-dev libatlas-base-dev gfortran libpng-dev libfreetype6-dev python-tk r-base
```

Comment:
Can people use libpng-dev on older versions of Ubuntu or would they still need to use libpng12-dev?